File: miniproj/compileandrunbench/codepow.cpp
#include <iostream>
#include <math.h>
int main()
{
	for(long int x = 1; x <= pow(99,4); x++){ // for x in range(1,99*4)
		std::cout << "X is " << x << std::endl; // send x to cout
		long double y = pow(x,2); // y = x ** 2
		std::cout << "Y is " << y << std::endl; // send y to cout
		long double z = pow(x,9); // z = y ** 9
		std::cout << "Z is " << z << std::endl; // send z to cout
		long double a = 3.14159 * z; // a = pi * z
		std::cout << "A is " << a << std::endl; // send a to cout
		long double tempb = a / x; // tempb = a div x
		long double b = floor(tempb); // b = floor tempb
		std::cout << "B is " << b << std::endl; // send b to cout
		long double c = fmod(a,x); // c is a mod x
		std::cout << "C is " << c << std::endl; // send c to cout
	}
}

File: miniproj/compileandrunbench/codepow.py
#!/usr/bin/python3
import math # import math
for x in range(1,99**4): # for loop 1 and 99 pow 4
    print(f'X is {x}') # print x 
    y = x ** 2 # y is x squared
    print(f'Y is {y}') # print y
    z = y ** 9 # z is y power 9
    print(f'Z is {z}') # print z
    a = math.pi * z # a is pi * z
    print(f'A is {a}') # print a
    b = a // x # b is a floor x
    print(f'B is {b}') # print b
    c = a % x # c is a mod x
    print(f'C is {c}') # print c
